spec/Vehicle/Vehicle.vspec

```diff
@@ -392,6 +392,11 @@ CurrentLocation:
   type: branch
   description: The current latitude and longitude of the vehicle.
 
+CurrentLocation.Timestamp:
+  datatype: string
+  type: sensor
+  description: Timestamp for latest position, formatted according to ISO 8601.
+
```

Comment:
add to the description, that it's the timestamp coming from the gps system.

Reply:
afterwards it can be merged, meeting 2021-12-14 approved

Reply:
Now updated, but will likely need rebase after #382 has been merged